### PR TITLE
Added has_password funtion in utils folder

### PR DESCRIPTION
**Pull request by Aman1143**
Issue: 10

#### Short description of what this resolves:
I just install bcryptjs module . Taking password from user as a req.body and hash it.

hash_password

### CODE REVIEW

**Review by BuddyLongLegs — CHANGES_REQUESTED** [review]
`password` instead of being extracted from body, should the and only argument of the function, return the hashPassword (replace hasPassword with hashPassword) string. Also i guess 12 is your encryption key (salt), set an enviornment variable `SECRET` to 12 and use process.env['SECRET'] instead of 12 in the hash function.

**Comment by Aman1143** [comment]
ok

**Comment by Aman1143** [comment]
I have changed that you want .
hash_password1

**Review by BuddyLongLegs — CHANGES_REQUESTED** [review]
Use `process.env.SECRET` (i.e., enviornment variable should be `SECRET`). Also just return the hashPassword (replace hasPassword with hashPassword). Returning password=hashPassword will cause an issue.

**Review by BuddyLongLegs — CHANGES_REQUESTED** [review]
Do not make hash_password the default export. Just put `export hash_password;`. Sorry for not mentioning this change earlier.

**Comment by Aman1143** [comment]
make change
hash_password2

File: utils/password.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import bcrypt from 'bcryptjs';
+import dotev from 'dotenv';
+
+dotev.config()
+
+const hash_password=(password)=>{
+     const hasPassword=bcrypt.hash(password,process.env.SECRET);
+    return  hasPassword;
+}
+export default hash_password;
```
Comment (BuddyLongLegs) [inline utils/password.js:10]: make it `export hash_password;`

**Comment by Aman1143** [comment]
I remove default that you want .

**Review by BuddyLongLegs — APPROVED** [review]
Great Work!

**Pull request merged**